python-sdk/tutorial.ipynb

```diff
@@ -403,9 +403,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "nuenv",
```

Comment:
Revert these changes probably. In the virtual environment installation instructions, we name the environment as `nuscenes`.